kore/src/Kore/Repl/Data.hs

```diff
@@ -290,7 +290,8 @@ helpText =
     \help                                     shows this help message\n\
     \claim [n|<name>]                         shows the nth claim, the claim with\
                                               \ <name> or if used without args\
-                                              \ shows the currently focused claim\n\
+                                              \ shows the currently focused claim\
+                                              \ in the form: LHS => (Operator) RHS \n\
```

Comment:
Would "modality" be more specific than "operator"? Or is "modality" not quite the right word?

Reply:
I think it is